src/components/commerce/OrderTable.tsx

```
@@ -0,0 +1,130 @@
+                  <div className="text-xs text-gray-500">{order.items} items</div>
+                )}
+              </td>
+              {showActions && (
+                <td className="px-6 py-4 whitespace-nowrap text-sm text-gray-500">
+                  <div className="flex space-x-2">
+                    {onView && (
+                      <button
+                        onClick={() => onView(order.id)}
+                        className="text-blue-600 hover:text-blue-700"
+                        title="View order"
+                      >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)